frontend/src/__tests__/LandingPage.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { LandingPage } from '../LandingPage'
+
+describe('LandingPage', () => {
+  it('renders', () => {
+    mount(<LandingPage />)
```

Comment:
`mount` returns a wrapper which would be truthy, so the test should pass, as long as the component is properly exported and renders. I guess at this early stage "it can be imported and successfully renders anything at all" is a reasonable test. [`shallow`](https://stackoverflow.com/a/38747914/130006) might be a faster way to accomplish the same task.